docs/azure-saas-docs/content/en/components/Identity/identity-framework-setup-manual.md

```diff
@@ -0,0 +1,392 @@
+
+- Upon being created, your Azure AD B2C Tenant will create a federated identity with your Azure subscription account.
+- Click on `Open B2C Tenant` quicklink to immediately change directory to your new Tenant
+
+### Step 3. (Optional) Invite collaborators to the new Tenant
+
+Note: Collaborators are other developers you wish to help manage your services, not customers
+
+![(Tenant User Management Screenshot)](/azure-saas/images/identity-framework-manual-step-3-invite-collaborators.png)
+
+- Go to the `Azure AD B2C` dashboard while logged into your new Tenant, click `Users` in the lefthand navbar
+- After navigating, you may create or invite guest users into your Tenant allowing them access
+
+### Step 4. Create necessary app registrations in Azure AD B2C Tenant
+
+![(Registered Apps Screenshot)](/azure-saas/images/identity-framework-manual-step-4-app-registration.png)
+
+- From the Tenant `Azure AD B2C` dashboard, click `App registrations`
+- Add new registrations corresponding to the following list of modules
+
+#### Modules
+
+- asdk-admin-api
+  - Display Name: asdk-admin-api
+  - Account Type: Accounts in any identity provider or organizational directory
+  - Grant Admin Consent to openid and offline_access_permissions: true
+- asdk-signupadmin-app
+  - Display Name: asdk-saas-app
+  - Account Type: Accounts in any identity provider or organizational directory
+  - Redirect URI: [Single-page application] https://{your-signupadmin-app-url}.com/signin-oidc
+  - Grant Admin Consent to openid and offline_access_permissions: true
+- asdk-saas-app
+  - Display Name: asdk-saas-app
+  - Account Type: Accounts in any identity provider or organizational directory
+  - Redirect URI: [Single-page application] https://{your-app-url}.com/signin-oidc
+  - Grant Admin Consent to openid and offline_access_permissions: true
+- asdk-permissions-api
+  - Display Name: asdk-permissions-api
+  - Account Type: Accounts in any identity provider or organizational directory
+  - Grant Admin Consent to openid and offline_access_permissions: true
+- IdentityExperienceFramework
+  - Display Name: IdentityExperienceFramework
+  - Account Type: Accounts in any identity provider or organizational directory
+  - Grant Admin Consent to openid and offline_access_permissions: true
+- ProxyIdentityExperienceFramework
+  - Display Name: ProxyIdentityExperienceFramework
+  - Account Type: Accounts in any identity provider or organizational directory
+  - Grant Admin Consent to openid and offline_access_permissions: true
+
+#### Module Client Secrets
+
+![(Client Secrets Screenshot)](/azure-saas/images/badgemeup-screenshot.png)
+
+- From the `App registration` view
+- For the apps listed below, navigate to them, then navigate to `Certificates & secrets` in their navbars
+- Add a new client secret for later reference, giving it an appropriate expiration and description
+- Modules Requiring Client Secrets:
+  - asdk-saas-app
```

Comment:
Let's move these requirements into the list above. This section should just be the instructions.